errors/error.go

```diff
@@ -22,6 +22,8 @@ type LoggableError struct {
 	err error
 }
 
+type FlagError error
+
```

Comment:
We have both `internal/errors.go` and this `errors/error.go`. Do we need both?

Reply:
Yes, this separation exists because of cyclic imports.

Reply:
`errors/error.go` doesn't seem to import anything other than `fmt`. How does that cycle occur?

Reply:
By creating another package for constants, I managed to get them compiled https://github.com/hazelcast/hazelcast-commandline-client/pull/61/commits/de27ebfb250fc6b9f182851abf7be8a1b9e04809